Lusas_Adapter/CRUD/Create/Loads/GravityLoad.cs

```diff
@@ -55,7 +56,14 @@ private IFLoadingBody CreateGravityLoad(GravityLoad gravityLoad, IFGeometry[] lu
             else
             {
                 lusasGravityLoad = d_LusasData.createLoadingBody(gravityLoad.Name);
-                lusasGravityLoad.setBody(gravityLoad.GravityDirection.X, gravityLoad.GravityDirection.Y, gravityLoad.GravityDirection.Z);
+                lusasGravityLoad.setBody(gravityLoad.GravityDirection.X * 9.80665, gravityLoad.GravityDirection.Y * 9.80665, gravityLoad.GravityDirection.Z * 9.80665);
+                if (!
+                    ((9.8 < lusasGravityLoad.getValue("accX") && lusasGravityLoad.getValue("accX") < 10.0 || lusasGravityLoad.getValue("accX") == 0.0)
+                    || (9.8 < lusasGravityLoad.getValue("accY") && lusasGravityLoad.getValue("accY") < 10.0 || lusasGravityLoad.getValue("accY") == 0.0)
+                    || (9.8 < lusasGravityLoad.getValue("accZ") && lusasGravityLoad.getValue("accZ") < 10.0 || lusasGravityLoad.getValue("accZ") == 0.0)))
+                {
+                    Engine.Base.Compute.RecordWarning($"The the gravitational acceleration of {lusasGravityLoad.getName()} is outside of the range 9.8 to 10.0 m/s^2.");
+                }
```

Comment:
```suggestion
                double g = 9.80665;
                lusasGravityLoad.setBody(gravityLoad.GravityDirection.X * g, gravityLoad.GravityDirection.Y * g, gravityLoad.GravityDirection.Z * g);
                }
```

I do not think we need to check if the users values are sensible here - if they want to put a gravitational factor of 0.17 to model structures on the moon, who are we to tell them otherwise?

As for g, you may consider just having a global variable m_g that is set in the Adapter - could be an AdapterSetting input that defaults to 9.806.

Reply:
Okay. I  made the decision to raise a warning if the value seemed out of the ordinary but let everything run of course. But I do agree as well. 

Sounds good. I tried to find a way of get a value for g from Lusas but could not. Sounds like a good use of the newly updated `AdapterSettings`.